org.gnome.Evolution.json

```diff
@@ -0,0 +1,381 @@
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://downloads.sourceforge.net/project/gtkspell/3.0.9/gtkspell3-3.0.9.tar.xz",
+					"sha256": "a4f4a4a2789f7499563e26d96b22d8085222ebe278da47d026b2de782b8b4d26"
+				}
+			]
+		},
+
+		{
+			"name": "evolution",
+			"buildsystem": "cmake",
+			"config-opts": [
+				"-DENABLE_AUTOAR=OFF",
+				"-DENABLE_CONTACT_MAPS=OFF",
+				"-DENABLE_GNOME_DESKTOP=OFF",
+				"-DENABLE_GTK_DOC=OFF",
+				"-DENABLE_INSTALLED_TESTS=OFF",
+				"-DENABLE_PST_IMPORT=OFF",
+				"-DENABLE_TEXT_HIGHLIGHT=OFF",
+				"-DENABLE_YTNEF=OFF",
+				"-DWITH_GLADE_CATALOG=OFF",
+				"-DWITH_HELP=OFF"
+			],
+			"cleanup": [
+				"/etc/xdg",
+				"/share/GConf"
+			],
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://gitlab.gnome.org/GNOME/evolution.git",
+					"branch": "gnome-3-30"
+				},
+
+				{
+					"type": "shell",
+					"commands": [
+						"cp config.h.in config.h.in.orig",
+						"cat config.h.in.orig | sed -e \"s|\\@VERSION_SUBSTRING\\@| \\(flatpak git$(git log --pretty=format:%h HEAD^..HEAD)\\)|\" >config.h.in"
+					]
+				},
+
+				{
+					"type": "script",
+					"commands": [
+						"export GIO_USE_NETWORK_MONITOR=base",
+						"echo evolution-flatpak | /app/bin/gnome-keyring-daemon --daemonize --unlock &",
+						"/app/libexec/dconf-service &",
+						"sleep 1",
+						"gsettings reset org.gnome.evolution-data-server network-monitor-gio-name",
+						"/app/libexec/evolution-source-registry &",
+						"sleep 1",
+						"/app/libexec/evolution-addressbook-factory -r &",
+						"/app/libexec/evolution-calendar-factory -r &",
+						"sleep 1",
+						"/app/bin/evolution.bin",
+						"sleep 3"
+					],
+					"dest-filename": "flatpak-evolution-wrapper.sh"
+				},
+
+				{
+					"type": "script",
+					"commands": [
+						"/app/libexec/evolution-data-server/evolution-dbus-session \\",
+						"	--exec /app/bin/evolution-wrapper \\",
+						"	--service-dir /app/share/dbus-1/services/ \\",
+						"	--iface org.freedesktop.Notifications:/org/freedesktop/Notifications \\",
+						"	--iface org.freedesktop.portal.Desktop:/org/freedesktop/portal/desktop \\",
+						"	--iface org.gnome.OnlineAccounts:/org/gnome/OnlineAccounts"
+					],
+					"dest-filename": "flatpak-evolution-run.sh"
+				}
+			],
+			"post-install": [
+				"mv /app/bin/evolution /app/bin/evolution.bin",
+				"cp flatpak-evolution-run.sh /app/bin/evolution",
+				"cp flatpak-evolution-wrapper.sh /app/bin/evolution-wrapper"
+			]
+		},
+
+		{
+			"name": "libmspack",
+			"cleanup": [ "/bin" ],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://www.cabextract.org.uk/libmspack/libmspack-0.6alpha.tar.gz",
+					"sha256": "1edbee82accb28e679ab538f803aab7a5a569e4102ccf1715b462b1bd915f921"
+				}
+			]
+		},
+
+		{
+			"name": "evolution-ews",
+			"buildsystem": "cmake",
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://gitlab.gnome.org/GNOME/evolution-ews.git",
+					"branch": "gnome-3-30"
```

Comment:
Is this a tag or a branch? If the latter, specify also a commit hash for builds reproducibility.